src/graphql/schema.ts

```diff
@@ -29,6 +25,11 @@ export const typeDefs = `#graphql
         rememberMe: Boolean!
     }
 
+    type Query {
+        hello: String
+        user(id: ID!): User
```

Comment:
```suggestion
        user(id: ID!): User!
```

Considerando que ao não encontrar um usuário vc retorna erro, podemos tipar como obrigatório sem problemas.